lib/userFactory.js

```diff
@@ -65,6 +65,38 @@ class UserFactory extends BaseFactory {
     }
 
     /**
+     * Get a user model by name, id, or token
+     * @method get
+     * @param   {Mixed}     config
+     * @param   {String}    [config.token]      Token of the user to look up
```

Comment:
I think it's okay not to specify config.username, you can also pass in `id` to get a user model. Technically you can pass in any valid key here to retrieve the model.